build.gradle

```diff
@@ -100,6 +100,22 @@ sourceSets {
         }
         resources.srcDir file('src/pactTest/resources')
     }
+    integrationTest {
+        java {
+            compileClasspath += main.output
+            runtimeClasspath += main.output
+            srcDir file('src/integrationTest/java')
+        }
+        resources.srcDir file('src/integrationTest/resources')
```

Comment:
is it worth renaming the actual tests to 
SecurityConfigurationIT
I see thats what NFDIV has + it makes more sense when searching

Reply:
Just did it.